src/util/fsUtil.js

```diff
@@ -28,4 +28,12 @@ module.exports = {
     path.dirname(normalizedFilename),
     path.basename(normalizedFilename, path.extname(normalizedFilename)) + ext,
   ),
+
+  removeExtension: (filePathWithExt) => {
```

Comment:
Just wondering - would a file path with `.` in the path but not the file itself pose a problem? (e.g. `~/.folderWithDot/fileWithNoExt`)

Would doing the check on the last item in the path, rather than the entire path itself, resolve this problem?

Reply:
Could `removeExtension` be written as:

```js
removeExtension: (filePathWithExt) => path.join(
	path.dirname(filePathWithExt),
	path.basename(filePathWithExt, path.extname(filePathWithExt)),
),
```
Should achieve the same effect, but using standard library components. `setExtension` can then be refactored to use `removeExtension`.

Reply:
> `setExtension` can then be refactored to use `removeExtension`.

@amad-person Do refactor the code as suggested, thanks!